### PR TITLE
script to launch watchtower

### PR DESCRIPTION
**Pull request by pgwillia**
## Context

We want to start practicing Continuous Deployment.  Watchtower is a docker container that polls docker hub for updated images for the containers that are running locally.  This script will be responsible for running the watchtower container.

Related to #1722

## What's New
Added watchtower script and using predeploy script to retrieve it.

### CODE REVIEW

File: bin/watchtower [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## this file comes from the Jupiter project  "https://raw.githubusercontent.com/ualbertalib/jupiter/${branch_name}/bin/watchtower"
+## it will start the watchtower container which polls docker hub for the latest image of the running docker containers
+## to review watchtower activity use `docker logs watchtower`
+
+docker stop watchtower && docker rm watchtower
+
+docker run -d \
+    --name watchtower \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    containrrr/watchtower \
```
Comment (murny) [inline bin/watchtower:10]: is that a spelling mistake? `containrrr`? Or that's on purpose?

Comment (pgwillia) [reply bin/watchtower:10]: That's on purpose https://hub.docker.com/r/containrrr/watchtower

File: bin/watchtower [hunk 2]
```diff
@@ -0,0 +1,12 @@
+## this file comes from the Jupiter project  "https://raw.githubusercontent.com/ualbertalib/jupiter/${branch_name}/bin/watchtower"
```
Comment (murny) [inline bin/watchtower:1]: We should maybe ensure we have a shebang comment at the top of this file as I assume this is a bash executable file?

`#!/usr/bin/env bash`

File: bin/watchtower [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+## to review watchtower activity use `docker logs watchtower`
+
+docker stop watchtower && docker rm watchtower
+
+docker run -d \
+    --name watchtower \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    containrrr/watchtower \
+    --interval 30 \
+    --cleanup
```
Comment (murny) [inline bin/watchtower:12]: Minor, typical best practice for all bash files should have a newline at the end of the file. As there's some weird things that can happen if you do not have this with how files get parse/etc (e.g if you use cat or diff on this file)

File: bin/watchtower (hunk 1 above)
Comment (murny) [reply bin/watchtower:10]: Haha sounds good 👍

**Comment by pgwillia** [comment]
Discovered a better approach.  Closing and pursuing that instead.

**Pull request closed without merging**